diff --git a/node_modules/react-native-navigation/lib/ios/RNNOverlayWindow.m b/node_modules/react-native-navigation/lib/ios/RNNOverlayWindow.m
index 744121a..9712339 100644
--- a/node_modules/react-native-navigation/lib/ios/RNNOverlayWindow.m
+++ b/node_modules/react-native-navigation/lib/ios/RNNOverlayWindow.m
@@ -1,11 +1,12 @@
 #import "RNNOverlayWindow.h"
+#import "RNNReactView.h"
 
 @implementation RNNOverlayWindow
 
 - (UIView *)hitTest:(CGPoint)point withEvent:(UIEvent *)event {
     UIView *hitTestResult = [super hitTest:point withEvent:event];
 
-    if ([hitTestResult isKindOfClass:[UIWindow class]] || [hitTestResult isMemberOfClass:UIView.class]) {
+    if ([hitTestResult isKindOfClass:[UIWindow class]] || ([hitTestResult.subviews count] > 0 && [hitTestResult.subviews[0] isKindOfClass:RNNReactView.class])) {
         return nil;
     }
 
